src/page/sign-in-step.page.ts

```diff
@@ -0,0 +1,19 @@
+import { $, ElementFinder } from 'protractor';
+
+export class SignInStepPage {
+  private emailTextField: ElementFinder;
+  private pwdTextField: ElementFinder;
+  private submitBtn: ElementFinder;
+
+  constructor () {
+    this.emailTextField = $('#email');
+    this.pwdTextField = $('#passwd');
+    this.submitBtn = $('#SubmitLogin > span');
+  }
+
+  public async performLogin(): Promise<void> {
+    await this.emailTextField.sendKeys('aperdomobo@gmail.com');
+    await this.pwdTextField.sendKeys('WorkshopProtractor');
+    await this.submitBtn.click();
```

Comment:
It is better if you receive the email and password as params, in that way the page object can be used to log in any person, not just for Alejandro Perdomo.

Reply:
Got it! where it should be better to define the params?